about.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+    href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+    rel="stylesheet"
+    integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+    crossorigin="anonymous"
+  />
+  <link
+    href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+    rel="stylesheet"
+  />
+  <link href="about.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Domine&family=Inter:wght@400;700&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Montserrat:wght@100&display=swap" rel="stylesheet"/> 
+    <title>About Conference Page</title>
+</head>
+<body>
+    <section class="forabt" >
+        <nav id = 'navbar'>
+        <div class="banner desktop"> 
+            <nav><a href="#"><img src="./assets/About/facebook.svg" alt="facebook-logo"></a></nav>
+              <nav><a href="#"><img src="./assets/About/twitter.svg" alt="twitter-logo"></a></nav>
+              <nav><a href="#">English</a></nav>
+              <nav><a href="#">My page</a></nav>
+              <nav><a href="#">Logout</a></nav>
+        </div>
+        <div class="toolbar">
+            <a id="hamburger" class=" hamburger" href="#"><img onclick="popmenu()" alt="Hambruger Icon" src="./assets/hamburger.svg"></a>
+            <a class = "logo" href="#"><img alt="Logo Icon" src="./assets/logo.svg"></a>
+            <div class="toolbar2 showcontent" id="visible" >
+                <div><img onclick ="popmenu() "src="./assets/Home/Icon - Cancel.png" alt=""></div>
+                <a class ="toollink" href="./about.html">About</a>
```

Comment:
- [ ] Hi, I think there is a mistake here that you can fix. Can you make it possible to navigate between the pages? there's a link from `Home` page to `About` page. Now you need to make the opposite(link from `About` to `Home` page).
- Also, Make sure the navigation works for Mobile size too.